spec/project_spec.rb

```diff
@@ -305,7 +305,13 @@ def expected
 
       dir = File.expand_path('../changelog', __dir__)
 
-      Dir["#{dir}/*.md"].each do |path|
+      Dir["#{dir}/*"].each do |path|
+        it 'has a file' do
```

Comment:
I think this would be better as

```suggestion
        it 'is a file' do
```
Although, doesn't this only overwrite a single spec every iteration? Doesn't the path need to appear in the spec name to generate a unique spec?